src/shared/components/resources/styles.module.scss

```diff
@@ -0,0 +1,25 @@
+.carouselButton {
+    color: #e0e0e0;
+
+    &.active {
+        color: black;
+        cursor: pointer;
+
+        &:hover {
+            opacity: 0.7;
+        }
+
+        &:active {
+            opacity: 0.9;
+        }
+    }
+}
+
+:global(.cbioportal-frontend) h1.blackHeader,
```

Comment:
lets but this in global.scss and also, i think it can be just .blackText.   I.e. not applied only to h1.    Possible it won't be strong enough and you would need to add h1....    You could also, try using existing bootstrap style .text-dark.  Again, not sure if it will overcome the h1 colors.